docs/reference/client-worker-configuration.md

```diff
@@ -31,19 +31,23 @@ The file may be called by any name but should have the following structure:
     "windows": {
       "localBuildDirectory": "C:\project\bin\windows"
     },
-    "macos": {...},
-    "linux": {...}
+    "macos": {
+      ...
+    },
+    "linux": {
+      ...
+    }
   }
 }
 ```
 
 | Field | Required/Optional | Description | 
 | :------------- | :------------- | :------- |
-| `workerType` | Required | Used to identify this worker type elsewhere in your project. |
-|`layer`| Optional | The name of the simulation layer of this worker, used to determine the load balancing strategy. For more information, see [introduction to layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/layers#introduction-to-layers).|
+| `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide between server and client workers. The `workerType` specified in this field must also match the `workerType` that your worker reports in its `ConnectionParameters` when connecting to a deployment. For more information, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |
```

Comment:
I'm not sure what "unique project-wide between server and client workers" means - is it that within a project, you can only use a particular `workerType` once (so if you use it for a client-worker, you can't use it for a server-worker)? If so I'd maybe leave out "between server and client workers" and just stick with "unique project-wide". 

I've also changed "that your worker reports" to that your worker instance reports" as I _think_ that is more accurate?

```suggestion
| `workerType` | Required | Used to identify this worker type elsewhere in your project. <br><br> **Note:** A `workerType` must be unique project-wide between server and client workers. The `workerType` specified in this field must also match the `workerType` that your worker instance reports in its `ConnectionParameters` when connecting to a deployment. For more information, see the [API reference](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connectionparameters-class). |
```

Reply:
Removed the "between server and client workers" part. Added "instance".